conjure-undertow-lib/src/main/java/com/palantir/conjure/java/undertow/lib/Attachments.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * (c) Copyright 2018 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.conjure.java.undertow.lib;
+
+import com.palantir.tokens.auth.UnverifiedJsonWebToken;
+import io.undertow.util.AttachmentKey;
+import java.util.Optional;
+
+public final class Attachments {
+
+    public static final AttachmentKey<Optional<UnverifiedJsonWebToken>> UNVERIFIED_TOKEN =
```

Comment:
The name 'TOKEN' is pretty generic, what about:

```suggestion
    public static final AttachmentKey<Optional<UnverifiedJsonWebToken>> UNVERIFIED_JWT =
```

Reply:
Good call